presentation/presentation.go

```diff
@@ -446,6 +446,10 @@ func (p *Presentation) Save(w io.Writer) error {
 			if err := zippkg.AddFileFromDisk(z, fn, img.Path()); err != nil {
 				return err
 			}
+		} else if img.Data() != nil && len(*img.Data()) > 0 {
```

Comment:
Could you please make this case the first one? Like this is done in the Word doc. I think keeping the same style everywhere would be best